src/AutoRest.CSharp.V3/Output/Builders/ClientBuilder.cs

```diff
@@ -265,17 +265,12 @@ private RestClientMethod BuildMethod(Operation operation, string clientName, IRe
                             bodyParameter = (requestParameter, constantOrReference);
                             break;
                         case ParameterLocation.Uri:
-                            if (defaultName == "$host")
```

Comment:
:+1:

Reply:
This was a workaround we no longer need.